controllers/eventingauth_controller_test.go

```diff
@@ -305,8 +324,18 @@ func deleteKubeconfigSecret(crName string) {
 	}
 	Eventually(func(g Gomega) {
 		err := k8sClient.Delete(context.TODO(), &kubeconfigSecret)
-		if err != nil {
-			g.Expect(kapierrors.IsNotFound(err)).Should(BeTrue())
-		}
+		g.Expect(kapierrors.IsNotFound(err)).Should(BeTrue())
+	}, defaultTimeout).Should(Succeed())
+}
+
+func reconcileEventingAuth(cr *eamapiv1alpha1.EventingAuth) {
+	By("Labelling EA to force reconcilation")
+	Eventually(func(g Gomega) {
+		var ea eamapiv1alpha1.EventingAuth
+		err := k8sClient.Get(context.TODO(), kpkgclient.ObjectKeyFromObject(cr), &ea)
+		g.Expect(err).To(BeNil())
+		ea.ObjectMeta.Labels = map[string]string{"reconcile": "now"}
```

Comment:
With the same label key/value the reconciliation will be triggered once, is this the intended behavior?